pkg/apis/config/v1alpha2/meshrootcertificate.go

```diff
@@ -136,11 +132,12 @@ type MeshRootCertificateConditionType string
 
 // MeshRootCertificateComponentStatuses is the set of statuses for each certificate component in the cluster.
 type MeshRootCertificateComponentStatuses struct {
-	Webhooks        MeshRootCertificateComponentStatus `json:"webhooks"`
-	XDSControlPlane MeshRootCertificateComponentStatus `json:"xdsControlPlane"`
-	Sidecar         MeshRootCertificateComponentStatus `json:"sidecar"`
-	Bootstrap       MeshRootCertificateComponentStatus `json:"bootstrap"`
-	Gateway         MeshRootCertificateComponentStatus `json:"gateway"`
+	ValidatingWebhook *bool `json:"validatingWebhook"`
```

Comment:
Per Keith's [proposal](https://gist.github.com/keithmattix/9ac73abbcb5721b0ce58102ac3ebff29), I thought the component statuses were going to be updated to the following: 

```go
type CertificateStatus struct {
  CurrentState CertificateStatusState
  QueuedTransition Transition
  Error string
}

type QueuedTransition struct {
  NewState CertificateStatusState
  At time.Time // An RFC 3339 formatted time. The certificateManager in `leaderMode` will perform the transition once it observes the MRC at a time after transition.At 
}
```